source/modules/assets/computers.rst

```diff
@@ -1,102 +1,289 @@
+Each item has its :doc:`own information <../../tabs/components>` (name, model, brand, memory capacity, number of cores/threads, etc.).
 
+Lines
+-----
 
-The different tabs
-------------------
+You can add telephone lines created in `Lines <../management/lines.html>`_
 
-.. include:: tabs/os.rst
 
-.. include:: tabs/components.rst
+Volumes
+-------
 
-.. include:: tabs/volumes.rst
+Summarises all the :doc:`volumes <../../tabs/volume>` present (hard disk, DVD) as well as the partitions present on the workstation
+(virtual disks such as Google Cloud may appear if they are installed as a network drive).
 
-.. include:: tabs/softwares.rst
+* Name
+* Automatic inventory (Yes /No)
+* partition
+* Mount point
+* File system
+* Global size
+* Free size
+* Free percentage
+* Encryptation (if the disk is encrypted, a padlock will be displayed)
 
-.. include:: tabs/connexions.rst
 
-.. include:: tabs/network-ports.rst
+Software
+--------
 
-.. include:: ../tabs/management.rst
+Lists all the :doc:`software <../../tabs/software>` brought up during the inventory and those added manually
 
-.. include:: ../tabs/contracts.rst
+It is possible to install (in the logical sense) software on a PC manually.
 
-.. include:: ../tabs/documents.rst
+To add new :doc:`software <../../modules/assets/softwares>`  to the list of applications, you need to go to the Assets > Software tab,
+which will then be visible from the software tab of the various elements of the installed base.
 
-.. include:: tabs/virtualization.rst
 
-.. include:: tabs/antivirus.rst
+Connections
+-----------
 
-.. include:: ../tabs/tickets.rst
+The :doc:`connections <../../tabs/connections>`  are all the other hardware connected to the machine :
 
-.. include:: ../tabs/problems.rst
+* `Device <devices.html>`_
+* `Monitor <monitors.html>`_
+* `Phone  <phones.html>`_
+* `Printers <printers.html>`_
 
-.. include:: ../tabs/changes.rst
+These items can be updated by the automatic inventory, but you can also connect them manually.
 
-.. include:: ../tabs/external-links.rst
 
-.. include:: ../tabs/notes.rst
+Network Ports
+-------------
 
-.. todo::
-   Fix this reference (include or link???)
-   **[Onglet "Réservations"](Les_différents_onglets/Onglet_Réservations.rst)**
-   Gestion des réservations pour un objet d'inventaire
+This tab allows to manage the :doc:`network ports <../../tabs/network_ports>` attached to an equipment.
+The information that can be viewed is:
 
-.. include:: ../tabs/historical.rst
+* Name
+* Port number
+* MTU
+* Speed
+* Internal status
+* Last change
+* Number of I/O bytes
+* Number of I/O erros
+* Duplex
+* VLAN
+* Connected to
+* Connection
+* Deleted
+
+
+Sockets
+-------
+
+:doc:`Sockets <../../tabs/sockets>` are the list of physical sockets present on the hardware. These sockets can be Ethernet, USB, HDMI, etc.
+This information cannot be returned by the automatic inventory, so you have to add it manually.
```

Comment:
I'm surprised by this sentence as glpi-agent can report a "ports" category which list discovered physical ports. This looks like it can help to report "Sockets". Can we have missed to integrate that data ?